libs/editor/WordPressEditor/src/main/java/org/wordpress/android/editor/GutenbergEditorFragment.java

```diff
@@ -98,17 +107,53 @@ private GutenbergContainerFragment getGutenbergContainerFragment() {
         return mRetainedGutenbergContainerFragment;
     }
 
+    public Bundle getTranslations() {
+        Bundle translations = new Bundle();
+        Locale defaultLocale = new Locale("en");
+        Resources currentResources = getResources();
+        Configuration configuration = currentResources.getConfiguration();
+        if (configuration.locale.equals(defaultLocale)) {
+            return translations;
+        }
+
+        DisplayMetrics metrics = new DisplayMetrics();
+        getActivity().getWindowManager().getDefaultDisplay().getMetrics(metrics);
+        configuration.locale = defaultLocale;
+        Resources defaultResources = new Resources(getActivity().getAssets(), metrics, configuration);
+
+        for (Field stringField : R.string.class.getFields()) {
+            int resourceId = 0;
+            try {
+                resourceId = stringField.getInt(R.string.class);
+            } catch (IllegalArgumentException | IllegalAccessException e) {
+                e.printStackTrace();
```

Comment:
in Android it's always better to Log the error and not print stack trace so that we can easily find and see what is the error about.